chapter_explainability/explainability-overview.md

```diff
@@ -111,81 +112,88 @@ class HeartDiseaseMLP(d2l.Classifier): #@save
-fig.colorbar(im, ax=ax.ravel().tolist(), location='bottom', fraction=0.046, pad=0.05)
-fig.suptitle('Weights of the hidden layers of the MLP classifier', y=0.01)
+        im = ax[i].matshow(param.detach().numpy())
+        ax[i].set_axis_off()
+        i += 1
+fig.colorbar(im, ax=ax.ravel().tolist(), location='right', fraction=0.02)
+fig.suptitle('Hidden layer weihghts of the MLP classifier', x=0.485, y=0.42)
 d2l.plt.show()
 ```
 
-Obviously, this figure does not deliver anything that is understandable and useful. Even a simple model like MLP is so difficult to understand, let alone more complex models such as convolutional network networks and Transformer.
-
 ## Explainability
 
-*Explainability* in machine learning is the degree to which humans can understand the decisions made by machine learning models. Often, simple models such as linear regression, logistic regression, and decision tree are easy to interpret (or self-explanatory) but fall short in capturing the intricate patterns in data, while complex models (also known as black-box models) sacrifice explainability for better performances. Good explanations are the key to communicating with practitioners and domain experts and enabling a wide deployment of machine learning algorithms. It also provides a new perspective to troubleshoot complex models, speedup debugging processes, bust biases and other potential AI potholes.
+In previous examples we tried to understand how simple and complex models work. 
+In machine learning, *explainability* is the degree to which humans can understand the decisions made by models. 
 
-Simple models can be valuable in some applications. That is also true in fields other than machine learning. Sometimes, people use easy-to-employ simple models to avoid cognitive overload. For example, emergency medical workers use the [rule of nines](https://en.wikipedia.org/wiki/Wallace_rule_of_nines) :numref:`fig_ruleof9s` to quickly assess a burn's percentage of the total skin;  Recreational divers use a [diving planner](https://en.wikipedia.org/wiki/Recreational_Dive_Planner) to determine how long they can safely stay underwater at a given depth to avoid putting too much nitrogen in their bodies which can cause injury or even death.
+Often, simple models such as linear regression, logistic regression, and decision trees are easy to interpret (or self-explanatory) but fall short in capturing the intricate patterns in data.
+Nonetheless,
+humans have been using easy-to-employ simple models to avoid cognitive overload. For example, emergency medical workers use the [rule of nines](https://en.wikipedia.org/wiki/Wallace_rule_of_nines) :numref:`fig_ruleof9s` to quickly assess a burn's percentage of the total skin; Recreational divers use a [diving planner](https://en.wikipedia.org/wiki/Recreational_Dive_Planner) to determine how long they can safely stay underwater at a given depth to avoid putting too much nitrogen in their bodies which can cause injury or even death.
```

Comment:
it seems that in this way, we are mixing the simple models in machine learning and other domains.

Reply:
This just shows simple models exist for a reason, regardless of domains